UI/window-basic-main.cpp

```diff
@@ -6639,8 +6641,8 @@ void OBSBasic::ToggleShowHide()
 
 void OBSBasic::SystemTrayInit()
 {
-	trayIcon.reset(new QSystemTrayIcon(QIcon(":/res/images/obs.png"),
-			this));
+	trayIcon.reset(new QSystemTrayIcon(QIcon::fromTheme("obs-tray",
+					QIcon(":/res/images/obs.png")), this));
```

Comment:
Style:  Indent with two tabs for a line continuation.

```suggestion
			QIcon(":/res/images/obs.png")), this));
```